src/Umbraco.Core/Configuration/Models/ContentImagingSettings.cs

```diff
@@ -17,17 +14,8 @@ public class ContentImagingSettings
             }
         };
 
-        public IEnumerable<string> ImageFileTypes { get; set; } = new[] { "jpeg", "jpg", "gif", "bmp", "png", "tiff", "tif" };
-
-        public IEnumerable<IImagingAutoFillUploadField> AutoFillImageProperties { get; set; } = DefaultImagingAutoFillUploadField;
+        public string[] ImageFileTypes { get; set; } = new[] { "jpeg", "jpg", "gif", "bmp", "png", "tiff", "tif" };
```

Comment:
Just so everyone knows. ImageSharp does not support Tiff yet and ImageProcessor is not supported on NET Core (And is a complete (_read dead_) project). 

Image format configuration should be bound to whatever the lowest common denominator of the various potential solutions available not hard coded.

Reply:
Agreed, that sounds better, but probably for something outside of this PR.  Perhaps could add to `IImageUrlGenerator` (the abstraction introduced for different "image processors"?

Reply:
Agree this value is very related to the implementation of `IImageUrlGenerator`

Reply:
Have created a new PR for this amend: https://github.com/umbraco/Umbraco-CMS/pull/8948